src/test/regress/expected/citus_local_tables.out

```diff
@@ -485,10 +485,175 @@ NOTICE:  executing the command locally: TRUNCATE TABLE citus_local_tables_test_s
 -- test vacuum
 VACUUM citus_local_table_1;
 VACUUM citus_local_table_1, distributed_table, local_table, reference_table;
+-- test drop
 DROP TABLE citus_local_table_1, citus_local_table_2, distributed_table, local_table, reference_table;
 NOTICE:  executing the command locally: DROP TABLE IF EXISTS citus_local_tables_test_schema.reference_table_xxxxx CASCADE
 NOTICE:  executing the command locally: DROP TABLE IF EXISTS citus_local_tables_test_schema.citus_local_table_2_xxxxx CASCADE
 NOTICE:  executing the command locally: DROP TABLE IF EXISTS citus_local_tables_test_schema.citus_local_table_1_xxxxx CASCADE
+-- test some other udf's with citus local tables
+CREATE TABLE citus_local_table_4(a int);
+SELECT create_citus_local_table('citus_local_table_4');
+ create_citus_local_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- should work --
+-- insert some data & create an index for table size udf's
+INSERT INTO citus_local_table_4 VALUES (1), (2), (3);
+NOTICE:  executing the command locally: INSERT INTO citus_local_tables_test_schema.citus_local_table_4_1504037 AS citus_table_alias (a) VALUES (1), (2), (3)
+CREATE INDEX citus_local_table_4_idx ON citus_local_table_4(a);
+NOTICE:  executing the command locally: CREATE  INDEX   citus_local_table_4_idx_1504037 ON citus_local_tables_test_schema.citus_local_table_4_1504037 USING btree (a )
+SELECT citus_table_size('citus_local_table_4');
+ citus_table_size
+---------------------------------------------------------------------
+             8192
+(1 row)
+
+SELECT citus_total_relation_size('citus_local_table_4');
+ citus_total_relation_size
+---------------------------------------------------------------------
+                     24576
+(1 row)
+
+SELECT citus_relation_size('citus_local_table_4');
+ citus_relation_size
+---------------------------------------------------------------------
+                8192
+(1 row)
+
+BEGIN;
+  SELECT lock_relation_if_exists('citus_local_table_4', 'ACCESS SHARE');
+ lock_relation_if_exists
+---------------------------------------------------------------------
+ t
+(1 row)
+
+  SELECT count(*) FROM pg_locks where relation='citus_local_table_4'::regclass;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+COMMIT;
+-- hide first column (relationId) as it might change
+SELECT part_storage_type, part_method, part_key, part_replica_count, part_max_size, part_placement_policy FROM master_get_table_metadata('citus_local_table_4');
+ part_storage_type | part_method | part_key | part_replica_count | part_max_size | part_placement_policy
```

Comment:
I wonder if we should hide `part_max_size` as well?

Reply:
It's calculated from a constant, so I don't think that we need to hide it

https://github.com/citusdata/citus/blob/a58a4395abec1e38d83629b66d1b33766c040897/src/backend/distributed/operations/node_protocol.c#L153

https://github.com/citusdata/citus/blob/a58a4395abec1e38d83629b66d1b33766c040897/src/backend/distributed/operations/node_protocol.c#L143

https://github.com/citusdata/citus/blob/a58a4395abec1e38d83629b66d1b33766c040897/src/backend/distributed/operations/node_protocol.c#L71

Reply:
this is a small thing, so lets keep it the way it is but my main point is that it seems like it doesn't add a value to the test and if the constant changes we will need to update the test.